src/assets/takeAway/takeAwayText.js

```diff
@@ -3,8 +3,9 @@ export default {
+            {paragraphText1: 'Emergency managers and water resource managers rely on the USGS’s water monitoring system and the associated water-data delivery to address complex water challenges involving too much, too little, or poor-quality water. Each year, floods, droughts, and water quality issues are a reminder of the vulnerability of the Nation’s physical and socioeconomic well-being and the importance of water monitoring. This monitoring system is currently functioning, but it was designed many decades ago to address 19th and 20th century challenges and needs major upgrades to meet the increasingly complex water challenges facing communities across the Nation.'},
+            {paragraphText2: 'In 2019, the USGS established a new initiative called the <a href="https://www.usgs.gov/mission-areas/water-resources/science/usgs-next-generation-water-observing-system-ngwos" target="_blank">Next-Generation Water Observing System (NGWOS)</a> to provide high-resolution, state-of-the-art, real-time data on water quantity and quality necessary to support modern water prediction and assessments while also serving as an incubator for new instrumentation and innovative methods of monitoring water resources. The USGS plans to implement NGWOS in at least 10 watersheds with diverse environmental settings that can serve to represent larger regions across the Nation. Data from these intensively monitored watersheds will be used (alongside data from existing monitoring networks) to help develop advanced models to fill in data and knowledge gaps that limit water prediction and assessments in those watersheds that can be extended to support predictions/assessments at a Nation scale. This advanced observing system will provide quantitative information on streamflow, evapotranspiration, snowpack, soil moisture, a broad suite of water quality constituents, connections between groundwater and surface water, and water use.'
```

Comment:
> to support predictions/assessments at a Nation scale

Nation --> National (and just generally, the sentence that this phrase appears in feels very long and it bit difficult to read)